File: test/fixtures/output/java/unirest/application-json.java
HttpResponse<String> response = Unirest.post("https://httpbin.org/anything")
  .header("content-type", "application/json")
  .body("{\"number\":1,\"string\":\"f\\\"oo\",\"arr\":[1,2,3],\"nested\":{\"a\":\"b\"},\"arr_mix\":[1,\"a\",{\"arr_mix_nested\":{}}],\"boolean\":false}")
  .asString();

File: src/targets/r/httr/fixtures/application-json.r
library(httr)

url <- "https://httpbin.org/anything"

payload <- "{\"number\":1,\"string\":\"f\\\"oo\",\"arr\":[1,2,3],\"nested\":{\"a\":\"b\"},\"arr_mix\":[1,\"a\",{\"arr_mix_nested\":{}}],\"boolean\":false}"

encode <- "json"

response <- VERB("POST", url, body = payload, content_type("application/json"), encode = encode)

content(response, "text")